source/query/collation.txt

```diff
@@ -0,0 +1,62 @@
+.. _query-bar-collation:
+
+=================================================
+Set Language Specific Rules for String Comparison
+=================================================
+
+.. default-domain:: mongodb
+
+.. admonition:: Available in |compass| Only
```

Comment:
I like this note, I just think the title of the note and the content of the note are kind of confusing / overkill. 

The title of the note says: "Available in MongoDB Compass Only" and the content says "...unavailable in the MongoDB Compass Community..." - I think that could confuse people (we're pushing users in two ways, at once). 

Could we change the title of the note to just say "Note" (or something else) and then have the content say "This feature is unavailable in MongoDB Compass Community Edition.", or "This feature is unavailable in MongoDB Compass Community Edition. It is only available in MongoDB Compass."

Should we link to the Compass index page in the note?